pkg/collector/corechecks/oracle-dbm/metadata.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build oracle
+
+package oracle
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/oracle-dbm/common"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/oracle-dbm/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type dbInstanceMetadata struct {
+	Dbm            bool   `json:"dbm"`
+	ConnectionHost string `json:"connection_host"`
+}
+
+type dbInstanceEvent struct {
+	Host               string             `json:"host"`
+	AgentVersion       string             `json:"agent_version"`
+	Dbms               string             `json:"dbms"`
+	Kind               string             `json:"kind"`
+	CollectionInterval uint64             `json:"collection_interval"`
+	DbmsVersion        string             `json:"dbms_version"`
+	Tags               []string           `json:"tags"`
+	Timestamp          float64            `json:"timestamp"`
+	Metadata           dbInstanceMetadata `json:"metadata"`
+}
+
+func sendDbInstanceMetadata(c *Check) error {
+	configTags := make([]string, len(c.config.Tags))
+	copy(configTags, c.config.Tags)
+	m := dbInstanceMetadata{
+		Dbm:            true,
+		ConnectionHost: config.GetConnectData(c.config.InstanceConfig),
+	}
```

Comment:
Do you have easy access to the agent version from the code here? We have `integration_version` for the other DBMSes because the integration version is separate from the agent and can be upgraded/installed independent of the agent version. But in this case, the `integration_version` _is_ tied to the agent so having the field set to the agent version might make sense.

Reply:
I can the `integration_version` to the agent version. I'll open a separate ticket for that.